starter_code/views/random-beers.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="theRandomBeerBody">
+    {{#each oneBeer}}
+    {{> beerPartial oneBeer}}
```

Comment:
instead of 'oneBeer' write 'this' which would reference to the 'oneBeer' in the each